Add Font Awesome 6 to devupdate (blocked by MDL-76989) (#569)
moodle/devdocs: docs/devupdate.md

```

</CodeExample>

## Font Awesome 6

The Font Awesome third-party library has been upgraded from 4.7 to 6.3.0 in MDL-76989. The syntax has slightly changed. The free version included in Moodle only supports the solid and regular styles:

```css
<!-- Version 4's syntax -->
<i class="fa fa-camera-retro"></i>
<!-- Version 6's syntax -->
<i class="fa-solid fa-camera-retro"></i>
```

> Font Awesome 6 is backwards compatible (because shim has been included too), so the old syntax is still working.

Reviewer: minor grammar: 

```suggestion
Font Awesome 6 is backwards compatible (because a shim has been included too), so the old syntax still works.
```

> More information about the changes between 4 and 6, like the icons that have been renamed, can be found in https://fontawesome.com/docs/web/setup/upgrade/upgrade-from-v4

Reviewer: ```suggestion
More information about the changes between 4 and 6, including any icons which have been renamed, can be found in https://fontawesome.com/docs/web/setup/upgrade/upgrade-from-v4
```
